src/Elm/Parser/Patterns.elm

```diff
@@ -273,10 +299,17 @@ recordPattern =
         |> Parser.Extra.continueWith
             (Core.map
                 (\commentsBeforeElements ->
-                    \elements ->
-                        { comments = Rope.flatFromList [ commentsBeforeElements, elements.comments ]
-                        , syntax = RecordPattern elements.syntax
-                        }
+                    \maybeElements ->
+                        case maybeElements of
+                            Nothing ->
+                                { comments = commentsBeforeElements
+                                , syntax = RecordPattern []
```

Comment:
Can be extracted to a top-level variable to avoid reallocating memory.

Reply:
the `RecordPattern []`? The function?

Reply:
I meant `emptyRecordPattern = RecordPattern []`

Reply:
done.